app/views/devise/registrations/new.html.erb

```diff
@@ -3,6 +3,31 @@
 <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
   <%= devise_error_messages! %>
 
+  <div class="field">
+    <%= f.label :first_name, 'First Name' %><br />
+    <%= f.text_field :first_name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :last_name, 'Last Name' %><br />
+    <%= f.text_field :last_name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :user_name, 'Username' %><br />
+    <%= f.text_field :user_name %>
+  </div>
+
+  <div class="field">
```

Comment:
I would hold off on this field for now until you can support the feature of uploading a user image.